server/src/main/scala/org/http4s/server/logging/ServerLogging.scala

```diff
@@ -0,0 +1,46 @@
+package org.http4s
+package server
+package logging
+
+import java.time.format.DateTimeFormatter
+import java.time.{Clock, OffsetDateTime}
+
+import org.http4s.headers.Authorization
+
+abstract class ServerLogging[F[_]] {
+  def enabled: Boolean
```

Comment:
Where is enabled called?  I couldn't find it via search.

Reply:
You're right, this can be removed. The intent was to let a caller inspect this property to avoid creating a log string but the signature of logRequestResponse was later changed to be completely side-effecting.